src/pkg/cli/client/grpc.go

```diff
@@ -90,7 +98,7 @@ func (g GrpcClient) Deploy(ctx context.Context, req *defangv1.DeployRequest) (*d
 
 		serviceInfos = append(serviceInfos, serviceInfo)
 	}
-	return &defangv1.DeployResponse{Services: serviceInfos}, nil
+	return &defangv1.DeployResponse{Services: serviceInfos}, warnings
```

Comment:
Will alway returns non-nil :( Let's get rid of `Warnings` for once and for all

Reply:
Fixed